src/components/main_page/GongZhongHao.js

```diff
@@ -1,8 +1,46 @@
 import React from "react";
+//import "../../styles/main_page/_GongZhongHao.scss";
+//import "../../styles/styles/style.css.map";
 
 export default function GongZhongHao(props) {
     return (
-        <div className="gzh-container">
+        // <div className="gzh-container">
+        // </div>
+        <div className="MainGrid">
```

Comment:
The class name is not very meaningful and specific. `gzh-container` in fact is a good one.

Reply:
https://github.com/UT-Applicataion-Development-Association/ADA-Website/pull/28/commits/bbcdde90e86d68ab6f2c7fffa7a41cbf79e487d3